### ISSUE TITLE
servicemonitor resource does not allow configuration of scrapeTimeout

### ISSUE THREAD

**Issue by cjwong1098** [qonto/prometheus-rds-exporter#13]
prometheus scraping this metrics endpoint is resulting in context deadline exceeded. Prometheus has a default of a scrape timeout of 10s. I think depending on how many rds instances in an account, this exporter may take longer than 10s to respond back. 

Can we add the scrape timeout param to servicemonitor to allow one to set the scrape timeout to a custom length.

example change to make in configs/helm/templates/servicemonitor.yaml
```
spec:
  jobLabel: app.kubernetes.io/name # Ensure prometheus job name match application's name
  endpoints:
    - path: /metrics
      port: http
      interval: {{ .Values.serviceMonitor.interval }}
      scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
```

**Comment by vmercierfr** [comment]
I submitted https://github.com/qonto/prometheus-rds-exporter/pull/18 to add `servicemonitor` parameters that could be relevant.

Can you take a look at the PR and let me know if you see any other parameters to customize?

**Comment by cjwong1098** [comment]
this looks good to me!

**Comment by vmercierfr** [comment]
We just released the [v0.2.6](https://github.com/qonto/prometheus-rds-exporter/releases/tag/v0.2.6) that contains a configurable `scrapeTimeout` parameter.
